resources/lib/ipwww_video.py

```diff
@@ -8,11 +8,12 @@
 import datetime
 import time
 import json
+import requests
```

Comment:
This import should no longer be required after the removal of the delete function.

Reply:
Thanks, done.